tools/c7n_azure/tests_azure/tests_resources/test_container_host.py

```diff
@@ -554,7 +583,8 @@ def test_unload_policy_file_with_bad_schema(self,
         with open(file_path, 'w') as f:
             f.write(policy_string)
 
-        host.unload_policy_file(file_path, {})
+        host.load_policy(file_path, host.policies)
```

Comment:
Was this test just no doing anything before since it didn't load the bad policy?

Reply:
The code path it was testing was the same I believe, but it was just luck with the implementation (loading a policy with a bad schema is almost the same as not loading one at all).  I fixed it to explicitly do what it says it is doing.